_examples/hello_world/main.go

```diff
@@ -0,0 +1,70 @@
+package main
+
+import (
+	"image/color"
+	"log"
+
+	"golang.org/x/image/font/gofont/goregular"
+
+	"github.com/ebitenui/ebitenui"
+	"github.com/ebitenui/ebitenui/widget"
+	"github.com/golang/freetype/truetype"
+	"github.com/hajimehoshi/ebiten/v2"
+)
+
+type game struct {
+	ui *ebitenui.UI
+}
+
+func main() {
+	ebiten.SetWindowSize(900, 800)
+	ebiten.SetWindowTitle("Ebiten UI Hello World")
+	ebiten.SetWindowResizingMode(ebiten.WindowResizingModeEnabled)
+
+	// This creates the root container for this UI.
+	// All other UI elements must be added to this container.
+	rootContainer := widget.NewContainer()
+
+	// This adds the root container to the UI, so that it will be rendered.
+	eui := &ebitenui.UI{
+		Container: rootContainer,
+	}
+
+	// This loads a font and creates a font face.
+	ttfFont, err := truetype.Parse(goregular.TTF)
+	fontFace := truetype.NewFace(ttfFont, &truetype.Options{
+		Size: 32,
+	})
+
+	// This creates a text widget that says "Hello World!"
+	helloWorldLabel := widget.NewText(
+		widget.TextOpts.Text("Hello World!", fontFace, color.White),
+	)
+
+	// To display the text widget, we have to add it to the root container.
+	rootContainer.AddChild(helloWorldLabel)
+
+	game := game{
+		ui: eui,
+	}
+
+	err = ebiten.RunGame(&game)
+	if err != nil {
+		log.Print(err)
+	}
+}
+
+func (g *game) Update() error {
+	// ui.Update() must be called in ebiten Update function, to handle user input and other things
+	g.ui.Update()
+	return nil
+}
+
+func (g *game) Draw(screen *ebiten.Image) {
+	// ui.Draw() should be called in the ebiten Draw function, to draw the UI onto the screen
```

Comment:
// It should also be called after all other rendering for your game so that it shows up on top of your game world.